README.md

```diff
@@ -117,6 +108,17 @@ Also, check [Danger's guide on Phrasing](http://danger.systems/js/usage/culture.
 
 - [x] Warn if CSS files were changed
 
+### Nice to have
+
+- [ ] (Platform/Language agnostic) Warn when Amazon Secret Key is hardcoded
+- [ ] (iOS) Warn if changes made in Cakefile may 'break' provisionings and sign certificates configurations
+- [ ] (iOS) Warn when ATS Exception is set in plist
```

Comment:
I think we should considering adding this one for iOS as a must instead of nice to have